client/lib/paths/login/index.js

```diff
@@ -121,5 +122,9 @@ export function login( {
 		url = addQueryArgs( { gravatar_flow: '1' }, url );
 	}
 
+	if ( pluginName ) {
+		url = addQueryArgs( { pluginName }, url );
```

Comment:
I'm unsure if this is picking the query args correctly, should it be the following instead? 🤔  
```suggestion
		url = addQueryArgs( { plugin_name: pluginName }, url );
```

Reply:
My bad 🤦 

Thank you for the catch!